File: QUESTÃO 6.py
E = []
O = []

while True:
    number = int(input("Enter a integer (or type to '000'): "))
    if number == 000:
        break
    elif number %2 == 0:
        O.append(number)
        print(f"The number {number} is odd.")
    elif number %2 != 0:
        E.append(number)
        print(f"The number {number} is even.")
    else:
        print(f"Invalid input")
print(f"Even numbers list: {E}\n Odd numbers list: {O}")

File: QUESTÃO 6.js
let E = [];
let O = [];

while (true) {
    let input = prompt("Enter a integer (or type '000' to exit)")

    if (input === "000") {
        break
    } 

    let number = Number(input)

    if (isNaN(number)){
        alert("Invalid input")
    } else if (number %2 === 0) {
        E.push(number)
        alert("The number " + number + " id even.")
    } else {
        O.push(number)
        alert("The number " + number + " is odd.")
    }
    
}

alert("Even number list: " + E + " \n Odd numbers list: " + O)
